leetcodePython/BinarySearch/bs_2602.py

```diff
@@ -0,0 +1,30 @@
+import bisect
+from typing import List
+
+
+class SolutionBruteForce:
+    def minOperations(self, nums: List[int], queries: List[int]) -> List[int]:
+        ans = []
+        for query in queries:
+            ops = 0
+            for num in nums:
+                ops += abs(num - query)
+            ans.append(ops)
+        return ans
+
+
+class SolutionBS:
+    def minOperations(self, nums: List[int], queries: List[int]) -> List[int]:
+        curr_sum = 0
+        nums.sort()
+        prefix_sum = [0]
```

Comment:
Need to review why prefix 0 for prefix sum.

For `query` smaller than all `nums`:
* `idx` = 0
* the smaller num operation is 0.

* `query * idx` = 0
* `prefix_sum[idx]` should also be 0